blogs/main.py

```diff
@@ -5,7 +5,10 @@
     UserResp, 
     ShowBlogs, 
     ShowUser,
-    Login
+    Login,
```

Comment:
can you follow the proper structure (different files and divide them into different layers) as you do for a real app.